### PR TITLE
T1 NFS Cluster creation

### PR DESCRIPTION
**Pull request by TejasC88**
Pass log : http://magna002.ceph.redhat.com/ceph-qe-logs/tejas/nfs_cluster.log

Signed-off-by: Tejas Chandramouli <tchandra@redhat.com>

### CODE REVIEW

File: rgw/v2/tests/nfs_ganesha/nfs_cluster.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+    # create user export for each user and bucket export for each bucket under
+    if config.test_ops["create_user_export"]:
+        for each_user in user_info:
+            auth = Auth(each_user, ssh_con, ssl=config.ssl)
+            rgw_conn = auth.do_auth()
+            uid = each_user["user_id"]
+            pseudo = f"/{uid}"
+            user_exp = utils.exec_shell_cmd(
+                f"ceph nfs export create rgw --cluster-id {cluster_id} "
```
Comment (vivekanandan-k-rh) [inline rgw/v2/tests/nfs_ganesha/nfs_cluster.py:76]: Can we move this part to reusables?

File: rgw/v2/tests/nfs_ganesha/nfs_cluster.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                    )
+                    buck = reusable.create_bucket(bucket_name, rgw_conn, each_user)
+                    pseudo_buck = f"/{bucket_name}"
+                    buck_exp = utils.exec_shell_cmd(
+                        f"ceph nfs export create rgw --cluster-id {cluster_id} "
+                        f"--pseudo-path {pseudo_buck} --bucket {bucket_name}"
+                    )
+
+    if config.test_ops["delete_cluster"]:
+        cluster_info = utils.exec_shell_cmd(f"ceph nfs cluster rm {cluster_id}")
```
Comment (vivekanandan-k-rh) [inline rgw/v2/tests/nfs_ganesha/nfs_cluster.py:92]: Same as above. Please move it to reusables

**Comment by TejasC88** [comment]
Updated Pass log : Pass log : http://magna002.ceph.redhat.com/ceph-qe-logs/tejas/nfs_cluster.log

**Review by udaysk23 — COMMENTED** [review]
Since all commands are handeled by exceptions, script will always pass even if single command fails to execute.
PLease correct me if I am wrong @TejasC88

**Comment by TejasC88** [comment]
> Since all commands are handeled by exceptions, script will always pass even if single command fails to execute. PLease correct me if I am wrong @TejasC88

Changed the try except handing , request review @udaysk23

**Review by udaysk23 — APPROVED** [review]
LGTM

**Pull request merged**